src/Contact_agent/Agenda.java

```diff
@@ -0,0 +1,47 @@
+package Contact_agent;
```

Comment:
by convention, in java package names are lowercase and one word, so could be contactagent